src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,42 @@
 /* eslint-disable */
-import React from 'react';
+import TodoItem from './TodoItem';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => {
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
```

Comment:
It's a good practice to use React's `React.PropsWithChildren` when you're using `React.FC`. This way, you can use the `children` prop if you need to.